packages/simpleserialize.com/package.json

```diff
@@ -17,7 +17,7 @@
     "@babel/runtime": "^7.14.6",
     "@babel/types": "^7.14.5",
     "@chainsafe/ssz": "workspace:^",
-    "@lodestar/types": "^1.7.2",
+    "@lodestar/types": "^1.14.0",
```

Comment:
> any reason not to install the latest version 1.17.0?

I picked 1.14.0 because the latest released change for Deneb `sszTypes.ts` is 1.14.0. https://github.com/ChainSafe/lodestar/commits/unstable/packages/types/src/deneb/sszTypes.ts . So having 1.17.0 or 1.14.0 doesn't make a difference